ChangeLog

```diff
@@ -1,5 +1,12 @@
 mbed TLS ChangeLog (Sorted per branch, date)
 
+= mbed TLS 2.xx.xx branch released xxxx-xx-xx
+
+Features
```

Comment:
It's not a feature of the library, but instead an extra test, so I'd rather list this under 'Changes'.

Reply:
Ok. Amended.

Reply:
Perfect. Thanks.